app/posts/manage-school-placements/2024-02-23-beta-location-analysis-phase-3.md

```diff
@@ -0,0 +1,161 @@
+| Phase of education | No | Yes |
+| Ofsted rating | No | No |
+| Religious character | No | No |
+
+## Leeds-based providers expanded network
+
+All the providers in Leeds also operate in Bradford, Calderdale, and Kirklees, so these are both excellent options for a second region.
+
+Bradford has many placement schools that are ‘Academies’, so this region has the best potential to improve the representativeness of the Leeds Placement schools.
+
+![Heatmap showing the number of placement schools in each local authority that hosted trainees from a provider located in Leeds. The lighter the colour, the greater the number of placement schools.](leeds-based-providers-expanded-network.png "Heatmap showing the number of placement schools in each local authority that hosted trainees from a provider located in Leeds. The lighter the colour, the greater the number of placement schools.")
+
+| Local authority | Number of providers in the area | Number of placements schools in the area |
+| --- | --- | --- |
+| Leeds | 6 | 217 |
+| Bradford | 6 | 137 |
+| Calderdale and Kirklees | 6 | 104 |
+| Wakefield | 3 | 82 |
+| North Yorkshire CC | 4 | 71 |
+| Barnsley, Doncaster and Rotherham | 5 | 33 |
+| East Riding of Yorkshire | 2 | 10 |
+| Sheffield | 2 | 7 |
+| Hartlepool and Stockton-on-Tees | 2 | 6 |
+| South Teeside | 2 | 6 |
+
+### Leeds and Bradford
+
+Including both Leeds and Bradford in the sample area makes the ‘Establishment type’ more representative of the wider placement school population. However, this area is not representative of the ‘Phase of education’ or ‘Percentage of free school meals’.
+
+Therefore, whilst this area does not improve the overall representativeness of the sample, it could be helpful if a larger sample of ‘Academies’ is required for targeted analysis.
+
+| Characteristic | Representative across all categories | Representative across categories of interest |
+| --- | --- | --- |
+| Pupil numbers | No | Not applicable |
+| Urban-rural setting | No | No |
+| Percentage of free school meals | No | Not applicable |
+| Establishment type | No | Yes |
+| Phase of education | No | No |
+| Ofsted rating | Yes | Yes |
+| Religious character | Yes | Yes |
+
+## Tyneside-based providers expanded network
+
+6 of the providers in Tyneside also operate in Northumberland and Durham.
+
+Both have similar characteristics, although Northumberland has a better distribution of rural-urban classification.
+
+![Heatmap showing the number of placement schools in each local authority that hosted trainees from a provider located in Tyneside. The lighter the colour, the greater the number of placement schools.](tyneside-based-providers-expanded-network.png "Heatmap showing the number of placement schools in each local authority that hosted trainees from a provider located in Tyneside. The lighter the colour, the greater the number of placement schools.")
+
+| Local authority | Number of providers in the area | Number of placements schools in the area |
```

Comment:
Tyneside based providers

Reply:
We decided not to change the heading as it relates to the section.